vite.config.js

```diff
@@ -19,11 +19,12 @@ export default defineConfig({
       fileName: format => `index.${format}.js`,
     },
     rollupOptions: {
-      external: ["react", "react-dom", "snakize", "react-dom/client"],
+      external: ["react", "react-dom", "react/jsx-runtime"],
```

Comment:
aligning with https://github.com/IgnacioNMiranda/vite-component-library-template/blob/main/vite.config.ts#L36
not sure why snakize was externalized